trino/client.py

```diff
@@ -770,19 +770,29 @@ def decorated(*args, **kwargs):
     return wrapper
 
 
+class NoOpRowMapper:
+    """
+    No-op RowMapper which does not perform any transformation
```

Comment:
nit: Might be useful to add - "Used when experimental_python_types is False." to the docstring.

Reply:
Done